test/CadlRanchProjects/azure/client-generator-core/access/src/Generated/Models/AbstractModel.cs

```diff
@@ -60,7 +60,7 @@ protected AbstractModel(string name)
         }
 
         /// <summary> Initializes a new instance of <see cref="AbstractModel"/>. </summary>
-        /// <param name="kind"> Discriminator. </param>
+        /// <param name="kind"> Discriminator property for AbstractModel. </param>
```

Comment:
There are such kind of description changes because TCGC updated the discriminator description in latest version.
Those changes are expected.